content/spin/v2/dynamic-configuration.md

```diff
@@ -27,10 +28,18 @@ when invoking the `spin up` command.
 
 The list of Spin features supporting runtime configuration includes:
 
-- [Application Variables](#variables-runtime-configuration)
-- [Key Value Storage](#key-value-store-runtime-configuration)
-- [SQL storage](#sqlite-storage-runtime-configuration)
-- [Serverless AI Compute](#llm-runtime-configuration)
+- [Application Variables Runtime Configuration](#application-variables-runtime-configuration)
```

Comment:
This section seems to have gotten replaced by the TOC?

Reply:
Ah, yes, hitting `ctrl` + `s` updates the ToC and makes a bit of a mess when there are 2 ToCs.
I have updated in the latest push.